gateway-ha/src/test/java/io/trino/gateway/ha/HaGatewayTestUtils.java

```diff
@@ -151,7 +150,8 @@ public static void setUpBackend(
                                 + active
                                 + ",\"routingGroup\": \""
                                 + routingGroup
-                                + "\"}");
+                                + "\"}",
+                        MediaType.parse("application/json; charset=utf-8"));
```

Comment:
Not wrong, but it's deprecated. 

https://stackoverflow.com/questions/57100451/okhttp3-requestbody-createcontenttype-content-deprecated
> Java Solution: Use create(String, MediaType) instead of create(MediaType, String) for example

Reply:
Haha .. what a weird refactor...